libraries/monitor/src/plugins/sentry.js

```diff
@@ -27,7 +27,7 @@ class SentryReporter {
   }
 
   report(error, level, extra) {
-    Sentry.configureScope(scope => {
+    Sentry.withScope(scope => {
```

Comment:
Ah this probably explains a lot of weirdness in our reports

Reply:
Especially the weird breadcrumbs :)